notification-hubs-sdk/src/test/java/com/microsoft/windowsazure/messaging/notificationhubs/TagEnricherTest.java

```diff
@@ -0,0 +1,79 @@
+package com.microsoft.windowsazure.messaging.notificationhubs;
+
+import android.content.Context;
+
+import androidx.test.core.app.ApplicationProvider;
```

Comment:
Correct me if I'm wrong, but I believed that tests like this that rely on Android provided types need to be in the `androidTest` source set.

Reply:
Answered above(robolectric thred)